cli/popper/commands/cmd_dot.py

```diff
@@ -27,17 +27,15 @@ def cli(ctx, wfile, recursive):
     """
     Creates a graph in the .dot format representing the workflow.
     """
-    def add_to_graph(graph_str, wf, parent, children):
+    def add_to_graph(graph_set, wf, parent, children):
         """Recursively goes through "next" and adds corresponding actions
         """
-        _parent = parent.replace(' ', '_').replace('-', '_')
         for n in children:
-            _n = n.replace(' ', '_').replace('-', '_')
-            graph_str += "  {} -> {};\n".format(_parent, _n)
+            graph_set.add('  "{}" -> "{}";'.format(parent, n))
```

Comment:
can we also make this a transparent box? something like:

```
[shape="box", style="filled", color="cyan", fillcolor="transparent"]
```

`cyan` seems to be a color that can be visualized on both dark and light terminal background colors.